File: src/main/java/com/uwetrottmann/trakt5/entities/Crew.java
package com.uwetrottmann.trakt5.entities;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Crew {

    public List<CrewMember> writing;
    public List<CrewMember> production;
    public List<CrewMember> directing;
    @SerializedName("costume & make-up")
    public List<CrewMember> costumeAndMakeUp;
    public List<CrewMember> art;
    public List<CrewMember> sound;
    public List<CrewMember> camera;

}

File: src/main/kotlin/com/redissi/trakt/entities/Crew.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
class Crew(
    var writing: List<CrewMember>? = null,
    var production: List<CrewMember>? = null,
    var directing: List<CrewMember>? = null,
    @Json(name = "costume & make-up")
    var costumeAndMakeUp: List<CrewMember>? = null,
    var art: List<CrewMember>? = null,
    var sound: List<CrewMember>? = null,
    var camera: List<CrewMember>? = null
)